src/components/link/link.tsx

```diff
@@ -57,13 +61,30 @@ export type EuiLinkProps = ExclusiveUnion<
   EuiLinkAnchorProps
 >;
 
-const EuiLink = forwardRef<HTMLAnchorElement | HTMLButtonElement, EuiLinkProps>(
+const externalLinkIcon = (
+  <EuiI18n token="euiLink.external.ariaLabel" default="External Link">
```

Comment:
Remember to use sentence case 

```suggestion
  <EuiI18n token="euiLink.external.ariaLabel" default="External link">
```